.eslintrc

```diff
@@ -1,25 +1,29 @@
 {
+    "extends": "eslint:recommended",
     "parser": "babel-eslint",
     "plugins": ["babel"],
     "rules": {
-        "strict": 0,
-        "indent": [2, 4],
-        "quotes": [2, "single"],
-        "linebreak-style": [2, "unix"],
+        "consistent-return": 2,
+        "eqeqeq": 2,
         "semi": [2, "always"],
-        "no-console": 0,
-        "babel/generator-star-spacing": 1,
-        "babel/new-cap": 1,
+
         "babel/array-bracket-spacing": 1,
+        "babel/arrow-parens": [1, "as-needed"],
+        "babel/new-cap": 1,
+        "babel/no-await-in-loop": 1,
         "babel/object-curly-spacing": 1,
-        "babel/object-shorthand": 0,
```

Comment:
I removed this because it defaulted to off as far as I can tell, so it is redundant to have it. I'd like to turn it on, but I'm fine leaving it as is.